test/e2e-tests-upgrade.sh

```diff
@@ -24,11 +24,27 @@
 # Scenario 2: install the previous release, create the pipelines and tasks, upgrade
 # to the current release, and validate whether the Tekton pipeline works.
 
+# Scenario 3: install the current pipelines, use the integration tests from the previous release(s)
```

Comment:
nit: in the text above, it says "there are two scenarios we need to cover in this script:", it should be updated to three